registries/_format/base64url.md

```diff
@@ -0,0 +1,136 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-5)
+base_type: string
+layout: default
+oas_version: < 3.1
+remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
```

Comment:
I'm going to suggest a bit of streamlining here and elsewhere after looking at the rendered page.  Do feel free to ignore!  I do not feel strongly about this.

> In OpenAPI 3.1, instead use [`contentEncoding: base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding), optionally alongside [`contentMediaType`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentmediatype)

I wouldn't say "or above" because it's generally implied and if we do ship a 3.2 the recommendation could change anyway.

Reply:
```suggestion
remarks: "When using OpenAPI 3.1 it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
```